script/server

```diff
@@ -167,10 +167,10 @@ then
   if [ -n "$redis_url" ]
   then
     redis_param="url: '$redis_url'"
-  fi 
+  fi
   if [ "$(bin/bundle exec ruby -e "require 'redis'; puts Redis.new($redis_param).ping" 2> /dev/null | grep -vE "is not writable|as your home directory temporarily" )" != "PONG" ]
   then
-    fatal "Can't connect to redis. Please check if it's running and if environment.redis is configured correctly in config/diaspora.yml."
+    fatal "Can't connect to redis. Please check if it's running and if environment.redis is configured correctly in config/diaspora.toml."
```

Comment:
What do you think of rewording? In the sense of just dropping "in ..." :D

Reply:
I think it's nice to point people to the correct file, but since they should know about the file from the installation guide, removing the "in ..." probably works too. :shrug: